packages/dbf/dbf.0.1.3/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A DBF/DBT files reader library"
+description:
+  "A DBF reader library with MEMO support and sequental record access"
+maintainer: ["Mykhayl Puzanov <pum@example.com>"]
+authors: ["Mykhayl Puzanov <pum@example.com>"]
+license: "GPL-3.0-or-later"
+tags: ["xBASE" "DBF" "data access"]
+homepage: "https://github.com/hothing/ocaml-libdbf"
+bug-reports: "https://github.com/hothing/ocaml-libdbf/issues"
+depends: [
+  "dune" {>= "3.17"}
+  "ocaml"
+  "odoc" {with-doc}
```

Comment:
```suggestion
  "cstruct"
  "bitstring"
  "ppx_bitstring"
  "ppx_cstruct"
  "odoc" {with-doc}
```

there is no need to re-release, but make sure to update your opam file in the repository with the missing dependencies above